apps/web/src/App.tsx

```diff
@@ -168,7 +171,19 @@ function App() {
                     <Route path="create" element={<CreateTenantPage />} />
                     <Route path=":identifier" element={<UpdateTenantPage />} />
                   </Route>
-                  <Route path={ROUTES.GET_STARTED} element={<GetStarted />} />
+                  <Route path={ROUTES.GET_STARTED} element={<GetStartedUseCase />}>
+                    <Route path="inapp" element={<GetStartedTab onboardingUseCase={onboardingUseCases.in_app} />} />
+                    <Route
+                      path="multichannel"
+                      element={<GetStartedTab onboardingUseCase={onboardingUseCases.multi_channel} />}
+                    />
+                    <Route path="digest" element={<GetStartedTab onboardingUseCase={onboardingUseCases.digest} />} />
+                    <Route path="delay" element={<GetStartedTab onboardingUseCase={onboardingUseCases.delay} />} />
+                    <Route
+                      path="translation"
+                      element={<GetStartedTab onboardingUseCase={onboardingUseCases.translation} />}
+                    />
```

Comment:
I'm wondering whether we need this duplication, the `onboardingUseCase` can be derived from `useParams`, and then we won't need to define any new route if we will have more use cases. But then we will need to handle each use case individually in `GetStartedTab`. They are similar but just the Digest is different a little. WDYT?

Reply:
My plan was to create a separate component for the digest because it's more complex. Having a distinct component would be simpler than adding complexity to the GetStartedTab and onboardingUseCases const.

We could possibly incorporate routing in the GetStartedTab for the digest. If we compare the two approaches: 5 routes in App vs 1 in GetStartedTab for digest.

I'll modify the route with parameters and assess the outcome. In the next cycle, we'll observe how the digest behaves.